modules/serverless-create-kafka-broker-custom-yaml.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+//  * serverless/knative_eventing/serverless-using-brokers.adoc
+
+[id="serverless-create-kafka-broker-custom-yaml_{context}"]
+= Creating a Kafka broker with custom configuration by using YAML
+
+You can create a Kafka broker by using YAML to create the `Kafka`-based broker object.
```

Comment:
```suggestion
You can create a Kafka broker by using YAML to create the Kafka-based broker.
```

The formatting goes either lowercase general reference, e.g. "broker", or is formatted using backticks if referring to a specific object, and is followed by the word object or resource, e.g. "the `KafkaChannel` resource. Guidelines are available here https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting